assets/js/custom/ProgramCredits.js

```diff
@@ -24,6 +24,8 @@ export function ProgramCredits (programId, defaultText, translationSaved, transl
   $(document).ready(function () {
     if (myProgram) {
       getLanguages()
+    } else {
+      programSection.showCustomTranslation(customTranslationApi, setCredits)
```

Comment:
Does this work?

```suggestion
      programSection.showCustomTranslation(customTranslationApi, credits.text)
```

Reply:
Based on some testing it doesn't look like it works. Good idea though, would have been nice if it worked